app/tests/test_ingest_hypothesis.py

```
@@ -0,0 +1,224 @@
+            expected_columns[category].update(columns)
+
+    if not category_files:
+        pytest.skip("No category files generated")
+
+    TECDownloader.consolidate_files()
+
+    for files in original_files.values():
+        for file_path in files:
+            assert not file_path.exists(), "Original parquet files should be removed after consolidation"
+
+    for category, files in category_files.items():
```

Ruff (0.14.7)

Loop control variable `files` not used within loop body

(B007)